github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -179,6 +179,37 @@ postsubmits:
                 memory: "1Gi"
               limits:
                 memory: "1Gi"
+    - name: publish-kubevirt-kubevirt.github.io-image
```

Comment:
Would you mind creating a build presubmit similar to [this one](https://github.com/kubevirt/project-infra/blob/e66f3b49d7c712a5aa1ff35f72c1142dce1690cc/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml#L63) for your image? This way we would catch failures in the image build before we merge.

Reply:
No problem, done.